src/system_locale.cpp

```diff
@@ -161,6 +163,12 @@ cata::optional<bool> UseMetricSystem()
     CFLocaleRef localeRef = CFLocaleCopyCurrent();
     CFTypeRef useMetricSystem = CFLocaleGetValue( localeRef, kCFLocaleUsesMetricSystem );
     return static_cast<bool>( CFBooleanGetValue( static_cast<CFBooleanRef>( useMetricSystem ) ) );
+#elif defined(__linux__)
```

Comment:
Is there a macro specific to glibc? This piece of code is also active with musl.

Reply:
Only if I include `feature.h`. But I'll follow the spec and check for empty string instead